xbmc/guilib/TextureBase.cpp

```diff
@@ -127,6 +127,7 @@ void CTextureBase::ClampToEdge()
       for (uint32_t x = imagePitch; x < texturePitch; x += blockSize)
         memcpy(dst + x, src, blockSize);
       dst += texturePitch;
+      src += texturePitch;
```

Comment:
Are you sure this shouldn't have been `src += imagePitch`?
(untested, but looking at the code seems to make more sense).

Reply:
Yes you are right. The change only works currently if image pitch and texture pitch are the same.

Reply:
As both work on m_pixels it was correct before. My comment is wrong.